app/bundles/PointBundle/Controller/AjaxController.php

```diff
@@ -44,11 +44,12 @@ public function getActionFormAction(Request $request, FormFactoryInterface $form
 
             if (isset($actions['actions'][$type])) {
                 $themes = ['@MauticPoint/FormTheme/Action/_pointaction_properties_row.html.twig'];
+
                 if (!empty($actions['actions'][$type]['formTheme'])) {
                     $themes[] = $actions['actions'][$type]['formTheme'];
                 }
 
-                $formType        = (!empty($actions['actions'][$type]['formType'])) ? $actions['actions'][$type]['formType'] : GenericPointSettingsType::class;
+                $formType        = (!empty($actions['actions'][$type]['formType'])) ? $actions['actions'][$type]['formType'] : null;
```

Comment:
Shorter syntax:
```suggestion
                $formType        = actions['actions'][$type]['formType'] ?? null;
```
Just a nice to have.